spec/formatting.md

```diff
@@ -320,16 +319,20 @@ Implementation-defined _functions_ SHOULD use an implementation-defined _namespa
 
 #### Option Resolution
 
-The result of resolving _option_ values is an unordered mapping of string identifiers to values.
+The result of resolving _option_ values is an unordered mapping of string identifiers to values,
+determined as follows:
 
-For each _option_:
+1. Let `res` be a new empty mapping.
+1. For each _option_:
+   1. Let `id` be the string value of the _identifier_ of the _option_.
+   1. Let `rv` be the _resolved value_ of the _option_ value.
+   1. If `rv` is a _fallback value_:
+      1. Set the value bound to `id` in `res` to an unresolved value.
+   1. Else:
+      1. Set the value bound to `id` in `res` to `rv`.
```

Comment:
See also line 330

```suggestion
      1. Bind the value `rv` to the ID `id` in `res`.
```

Reply:
Is there an existing expression for this sort of record/mapping/object property value assignment that we could crib from another spec? I'd prefer not inventing new language, and I'm drawing a blank for this.

Reply:
My best suggestion for a template would be the WHATWG Infra [Map](https://infra.spec.whatwg.org/#ordered-map).

Reply:
Infra Map is fine, but... ordered and not applicable in all programming environments.

Maybe just ignore my comment. My real comment was that "set the value bound to" is awkward.

Reply:
> My real comment was that "set the value bound to" is awkward.

That's the problem I'm also seeking to solve. Would this work? We don't use `[]` access in the other algorithms, but I would presume that it'd be pretty clear.
```suggestion
      1. Set `res[id]` to be `rv`.
```